build_config/lib/src/build_config.dart

```diff
@@ -170,6 +167,19 @@ class BuilderDefinition {
     this.buildTo,
     this.defaults,
   });
+
+  @override
+  String toString() => {
+        'target': target,
+        'autoApply': autoApply,
+        'import': import,
+        'builderFactories': builderFactories,
+        'buildExtensions': buildExtensions,
+        'requiredInputs': requiredInputs,
+        'isOptional': isOptional,
+        'buildTo': buildTo,
+        'defaults': defaults,
+      }.toString();
```

Comment:
I don't think this actually prints out that nicely (doesn't put each key/value on its own line). Consider JSON.encode or something like that?

Reply:
I don't think it has much impact. This is mostly to make test failures easy to parse and we should (hopefully) rarely need it use it